analyzers/its/regression-test.ps1

```diff
@@ -70,7 +70,6 @@ function Build-Project-MSBuild([string]$ProjectName, [string]$SolutionRelativePa
     $Env:PROJECT = $ProjectName
 
     Restore-Packages $msbuildVersion $solutionPath
-    # Note: Summary doesn't work for MSBuild 14
```

Comment:
This comment is not needed, we will not be able to pass "14.0" as a parameter anymore.